OpenTrons/opentrons-api
# fix(app): Fix RunFailed modal WhiteScreen and plus alpha

# Overview
This pr will fix run failed modal for desktop white screen issue
close RQA-1284
Update time format on Instruments to align with the hi-fi design 
[before]
![prev_timeformat](https://github.com/Opentrons/opentrons/assets/474225/adc545ca-e901-4ae1-a3bb-0b9acd33eeca)


[after]
This screenshot was taken before adding `UTC` label
![Screenshot 2023-08-22 at 6 31 17 PM](https://github.com/Opentrons/opentrons/assets/474225/3dff4993-13c8-4d2c-bf11-a029e123370e)

![Screenshot 2023-08-22 at 7 23 26 PM](https://github.com/Opentrons/opentrons/assets/474225/55b376b6-34c8-45c5-a653-85235e06fb2e)


timestamp with UTC label
https://www.figma.com/file/0kuPeCi1t2Auu2GPMnqRb2/Primary%3A-Opentrons-Flex-Touchscreen?type=design&node-id=18460-139711&mode=design&t=DJX4wAlMQiiPldKI-0



# Test Plan
- Go to All protocols
Check UTC label is displayed in ProtocolCard, Pinned Protocol 

- Go to ProtocolDetails
Check UTC label is displayed

- Go to RunSummary
Check UTC label is displayed and timestamp labels' layout aligns with the edited design (the above link)

- Go to Instruments -> one of Instruments (Last calibrated)
Check UTC label is displayed




# Changelog
- update ProtocolRunHeader
- update time format for instruments
- fix typo


# Review requests



# Risk assessment
low
